scripts/singleton/deploy.js

```diff
@@ -1,11 +1,10 @@
 const deployData = require('../../singleton/deploy.json')
-const { BN, toWei } = require('web3-utils')
 
 // Deploys a RelayHub instance, reading required data from singleton/deploy.json, the output of scripts/singleton/compute.sh
 // Call this file with truffle exec on the target network.
 async function deploy () {
   const funder = (await web3.eth.getAccounts())[0]
-  await web3.eth.sendTransaction({ from: funder, to: deployData.deployer, value: new BN(toWei('0.42', 'ether')) })
+  await web3.eth.sendTransaction({ from: funder, to: deployData.deployer, value: deployData.gas })
```

Comment:
If 'deployData.gas' is calculated with absolute precision (which is possible), this will not give enough by 21'000 gas units

Reply:
PS (for the record)
Hard-coding constructor parameters that are added now for differentiation between different chains with different opcode pricings, base currency cost, etc. defeats the purpose, so I propose deprecating this script entirely.